pywinauto/application.py

```diff
@@ -846,7 +845,7 @@ def connect(self, **kwargs):
             try:
                 timeout = float(kwargs['timeout'])
                 del kwargs['timeout']
-                wait_until_passes(self.timeout, Timings.app_connect_retry, partial(self.Connect, **kwargs), (ValueError))
+                time.sleep(timeout)

```

Comment:
Hey, what if an app can start from 5 to 30 seconds and you wait always for 35 seconds. It's very hard-coded way. You need to wait flexibly for better performance on fast machines. That why we have this task.

`time.sleep` can be called outside without any additional tricks inside.